lib/omnibus/packager.rb

```diff
@@ -67,13 +68,13 @@ def for_current_system
         family = "solaris"
       end
       if klass = PLATFORM_PACKAGER_MAP[family]
-        klass
+        [klass].flatten
       else
         log.warn(log_key) do
           "Could not determine packager for `#{family}', defaulting " \
           "to `makeself'!"
         end
-        Makeself
+        [Makeself].flatten
```

Comment:
But `Makeself` is a class and will never be an Array so the `#flatten` in this case will never be needed.